lib/benchpark/experiment.py

```diff
@@ -48,10 +49,141 @@ class Experiment(ExperimentSystemBase):
         Dict[str, benchpark.variant.Variant],
     ]
 
+    variant(
+        "scaling-factor",
+        default="2",
+        values=int,
+        description="Factor by which to scale values of problem variables",
+    )
+
+    variant(
+        "scaling-iterations",
+        default="4",
+        values=int,
+        description="Number of experiments to be generated",
+    )
+
```

Comment:
@scheibelp I have incorporated your earlier suggestions into the scaling methods. I also moved the entire scaling implementation directly into the base Experiment class. Please review and comment this is a good approach. Especially the fact that the base Experiment class will have two variants `scaling-factor` and `scaling-iterations` defined in it